test/retry_test.go

```diff
@@ -477,6 +533,79 @@ func (s) TestRetryStreaming(t *testing.T) {
 	}
 }
 
+func (s) TestMaxCallAttemptsStreaming(t *testing.T) {
```

Comment:
Can this be folded into the other test as another dimension to avoid repeating all the setup and test cases?

Reply:
folded into one test now